server/views/findInterventions/interventionDetails.njk

```diff
@@ -15,7 +15,8 @@
     <div class="govuk-grid-column-two-thirds">
       <h1 class="govuk-heading-xl">{{ presenter.title }}</h1>
 
-      <p class="govuk-body">{{ presenter.body }}</p>
+      <p class="govuk-body">{{ presenter.description }}</p>
+      <p class="govuk-body">{{ presenter.longDescription | striptags(true) | escape | nl2br }}</p>
```

Comment:
Are we going to need both descriptions now, or just the long description? I wonder if there's a clearer word than `description` to differentiate the two?

Reply:
I can't remember the conversation about this... Let me ask

Reply:
Ah, I think the intent is to have a single description and shorten it on the search results page but show the entire thing in the details.